flink-staging/flink-hbase/pom.xml

```diff
@@ -112,6 +112,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
Why did you add this dependency? 
There are no additional tests that would require it, right?